mayastor/src/subsys/opts.rs

```diff
@@ -43,7 +43,7 @@ impl Default for NexusOpts {
     fn default() -> Self {
         Self {
             replica_port: 8420,
-            nvmf_nexus_port: 4421,
+            nvmf_nexus_port: 8420, // Same target for now.
```

Comment:
don't we have the conflict then? If replica_port and nvmf_nexus_port are the same then we won't be able to create a nexus on a node with a replica, if my understanding is correct.

Reply:
I hit problems with the separate targets for nexus and replicas, and apparently the way they work also changes in the new SPDK, so we agreed we'll consider using separate targets for nexus and replicas to be 'blocked' on the SPDK update, but with this PR we'll unblock the CSI for nvmf work so that can get progress in the mean time,